docs/philosophy.md

```diff
@@ -0,0 +1,25 @@
+# Our philosophy
+
+## Why open-wc
+
+- The goal of Open Web Components is to empower everyone with a powerful and battle-tested setup for creating and sharing open source web components.
+- Javascript fatigue: Enjoy the peace of mind that comes from having a well-known default solution for almost everything. Open-wc has got you covered.
+- Making things accessible and approachable for people: Easy to use, ready to use, 'it just works' feeling
```

Comment:
potential rephrase: 
> Providing the "it just works" developer experience we all crave for the various aspects of the web components ecosystem.

Unless you define 'things'! :)

Reply:
yeah that sounds much better